manifest.json

```diff
@@ -63,7 +63,7 @@
           "angular-aria.min.js", "angular-aria.min.js.map"
         ],
         "jsFiles": [
```

Comment:
@seanlip  I think I will go with the later too, I remember on the design doc we had something like 
that. I am going to implement that.